website/src/_posts/2019-04-liftoff-10.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Day 10"
+date: 2019-04-04
+author: tyler
+image: "https://uppy.io/images/blog/30daystoliftoff/day10.jpg"
+series: 30 Days to Liftoff
+seriesSuffix: 'of 30'
+---
+
+Today marks the tenth day in our 30 Days to Liftoff! **April 25** is still in our sights as we work towards the **Uppy 1.0** release. The end of the working week is almost upon us but there's no sign of slowing down yet.  
+
+<center><img width="400" src="/images/blog/30daystoliftoff/day10.jpg"><br /></center>
+
+Things are looking good on our project board and the team has been working on tasks at a steady pace. A quick status update will give us a little insight into how things are going concerning all things Uppy.
+
+<!--more-->
+
+## In Progress
+
+- [Alex](https://github.com/nqst) and Artur are debugging [flickering on Drag & Drop](https://github.com/transloadit/uppy/pull/1400) that Chrome and Safari suffer from. Fingers crossed for a solution!
+
+- [Artur](https://github.com/arturi), working on the React Native side of things, has impressively added [file previews and tested hashes with different file types](https://github.com/transloadit/uppy/pull/988). He hopes to do a release once some testing is complete. In addition the React Native team has reschedulded their pair programming session to Friday, so expect some pretty significant updates tomorrow!
+
```

Comment:
Release is not a result of those particular changes, so maybe “after that he is panning to move on to releasing a new version of uppy”